### PR TITLE
Adjust description and add example

### PR DESCRIPTION
**Pull request by get-itips**
closes https://github.com/MicrosoftDocs/OfficeDocs-SharePoint-PowerShell/issues/132

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by dariomws — APPROVED** [review]
Should we keep the note formatted?

**Comment by get-itips** [comment]
> Should we keep the note formatted?

What's the last directive about that?

**Review by tseward — APPROVED** [review]

File: sharepoint/sharepoint-ps/sharepoint-online/Connect-SPOService.md [hunk 1]
```diff
@@ -137,7 +145,7 @@ The valid values are: Default | ITAR | Germany | China
 The default value is "default".
 
 > [!NOTE]
```
Comment (JohanFreelancer9) [inline sharepoint/sharepoint-ps/sharepoint-online/Connect-SPOService.md:147]:
```suggestion
```

File: sharepoint/sharepoint-ps/sharepoint-online/Connect-SPOService.md [hunk 2]
```diff
@@ -137,7 +145,7 @@ The valid values are: Default | ITAR | Germany | China
...
-> The ITAR value is for United States Government cloud tenancies only.  
+> The ITAR value is for GCC High and DoD tenancies only.
```
Comment (JohanFreelancer9) [inline sharepoint/sharepoint-ps/sharepoint-online/Connect-SPOService.md:148]:
```suggestion
**Note**: The ITAR value is for GCC High and DoD tenancies only.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
@dariomws, that is a valid question. @get-itips, because PowerShell does not support markdown, I suggest changing the format, please.

**Comment by get-itips** [comment]
> @dariomws, that is a valid question. @get-itips, because PowerShell does not support markdown, I suggest changing the format, please.

Certainly, but lately we've been accepting it in other PRs

Will change it :)

**Comment by dariomws** [comment]
@get-itips, it's a good question. Actually, I think the directive is not to use formatted notes on the office-docs-powershell repo.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Comment by tiburd** [comment]
@get-itips @dariomws @tseward @JohanFreelancer9 
Hi! 

Our admin, Thomas, said that alerts work in the repo when they are outside of the Synopsis section. I'll attend the PowerShell open office hours on Thursday to get complete guidance. I'll let you know and will update the wiki after that.

Thanks!

**Pull request merged**